tvb_storage/tvb/storage/storage_interface.py

```diff
@@ -0,0 +1,469 @@
+
+    def write_project_metadata_from_dict(self, project_path, meta_entity):
+        FilesHelper.write_project_metadata_from_dict(project_path, meta_entity, self.TVB_PROJECT_FILE)
+
+    def write_project_metadata(self, meta_dictionary):
+        self.files_helper.write_project_metadata(meta_dictionary, self.TVB_PROJECT_FILE)
+
+    def get_operation_folder(self, project_name, operation_id):
+        return self.files_helper.get_operation_folder(project_name, operation_id)
+
+    def remove_operation_data(self, project_name, operation_id):
+        self.files_helper.remove_operation_data(project_name, operation_id)
+
+    def remove_datatype_file(self, h5_file):
+        self.files_helper.remove_datatype_file(h5_file)
+        self.push_folder_to_sync(self.get_project_folder_from_h5(h5_file))
+
+    def move_datatype(self, new_project_name, new_op_id, full_path):
+        self.files_helper.move_datatype(new_project_name, new_op_id, full_path)
+
+    def get_images_folder(self, project_name):
+        return self.files_helper.get_images_folder(project_name, self.IMAGES_FOLDER)
+
+    def write_image_metadata(self, figure, meta_entity):
+        self.files_helper.write_image_metadata(figure, meta_entity, self.IMAGES_FOLDER)
+
+    def remove_image_metadata(self, figure):
+        self.files_helper.remove_image_metadata(figure, self.IMAGES_FOLDER)
+
+    def get_allen_mouse_cache_folder(self, project_name):
+        return self.get_allen_mouse_cache_folder(project_name)
+
+    def zip_folders(self, all_datatypes, project_name, zip_full_path, folder_prefix=""):
+        operation_folders = []
+        for data_type in all_datatypes:
+            operation_folder = self.get_operation_folder(project_name, data_type.fk_from_operation)
+            operation_folders.append(operation_folder)
+        FilesHelper.zip_folders(zip_full_path, operation_folders, folder_prefix)
+
+    @staticmethod
+    def zip_folder(result_name, folder_root):
+        FilesHelper.zip_folder(result_name, folder_root)
+
+    def unpack_zip(self, uploaded_zip, folder_path):
+        return self.files_helper.unpack_zip(uploaded_zip, folder_path)
+
+    @staticmethod
+    def copy_file(source, dest, dest_postfix=None, buffer_size=1024 * 1024):
+        FilesHelper.copy_file(source, dest, dest_postfix, buffer_size)
+
+    @staticmethod
+    def remove_files(file_list, ignore_exception=False):
+        FilesHelper.remove_files(file_list, ignore_exception)
+
+    @staticmethod
+    def remove_folder(folder_path, ignore_errors=False):
+        FilesHelper.remove_folder(folder_path, ignore_errors)
+
+    @staticmethod
+    def compute_size_on_disk(file_path):
+        return FilesHelper.compute_size_on_disk(file_path)
+
+    @staticmethod
+    def compute_recursive_h5_disk_usage(start_path='.'):
+        return FilesHelper.compute_recursive_h5_disk_usage(start_path)
+
+    # TvbZip methods start here #
+    def initialize_tvb_zip(self, dest_path, mode="r"):
```

Comment:
I think this method can be eliminated after a small refactor of its usages as we discussed